src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,80 @@
-# TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
+# TO-DO: Complete the selection_sort() function below
+def selection_sort(arr):
     # loop through n-1 elements
     for i in range(0, len(arr) - 1):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
+        # (hint, can do in 3 loc)
+        for j in range(i, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
 
+        # TO-DO: swap
+        temp = arr[smallest_index]
+        arr[smallest_index] = arr[i]
+        arr[i] = temp
```

Comment:
Ah! This is a great idea! I actually had the same idea when I worked through this. Python actually lets us do this on the fly, without having to declare a temp variable. DM me if you would like to know how to do this!